src/styles/typography.scss

```diff
@@ -12,6 +13,32 @@ $p-empty-state-height: 16px;
 $h6-empty-state-height: 12px;
 $small-empty-state-height: 12px;
 
+@function contain($list, $value) {
```

Comment:
copy paste from monday ui styles, we will export the helper functions in different pr (less urgent)